server/Routes/auth.js

```diff
@@ -0,0 +1,85 @@
+const express = require('express');
+
+const {
+    Client
+} = require('pg');
+const bcrypt = require('bcrypt');
+const client = new Client({
+    user: "postgres",
+    password: "123",
+    host: "localhost",
+    port: 6000,
+    database: "papel"
+});
+
+const router = express.Router();
+
+router.get('/', (req, resp) => {
+    resp.send("auth route is working")
+});
+
+//create user account
```

Comment:
you are supposed to use the MVC pattern. basically you should separate routes, models, controllers

but here I can see that the logic is in the same file as the route